plugins/kubernetes/app/models/kubernetes/role.rb

```diff
@@ -164,6 +164,7 @@ def role_config_file(reference, deploy_group:, project: project(), **args) # rub
       file = config_file
       if deploy_group && dynamic_folders?
         file = file.
+          sub('$deploy_group_permalink', deploy_group.permalink).
```

Comment:
it has to come before `$deploy_group` otherwise it will replace it to `pod998_permalink`